### PR TITLE
Pagination + ui tweaks

### PR DESCRIPTION
**Pull request by jpykie**
Feel free to reject this request if you want - I've just made some UI tweaks to make it more useful for myself and thought they may be good in the main branch

### CODE REVIEW

**Comment by oxan** [comment]
Thanks. Just a quick note that I'm interested in the feature, but haven't found time to review it yet. The pull request is highly appreciated though ;)

**Comment by oxan** [comment]
While I like the idea; I don't think this implementation is suitable to be included in mainline for several reasons:
- Ideally, I want pagination to be a universal (i.e. not skin-dependent) and (if the skin wants it) server-side function. If you have movie library with a 1000 movies, the current approach doesn't solve the real problems we are currently encountering: all movies are being loaded from the database, transfered from the service to WebMP and send to the client (along with their artwork), which is very slow. While the async image loading helps a bit on the last part, the other items can still take up several seconds.
- I want to keep the default skin simple, with preferably as few fancy JavaScript and CSS as possible (since no one on the team is comfortable maintaining complex client-side scripts, and doing it in a skin allows for more flexibility later on).
- This conflicts with ongoing reworks in other branches: we're going to be pulling all our JS libraries from NuGet very soon, and there's a rework in the progress to unify all lists views which would nicely support a more generic and performant pagination implementation.

So in conclusion, I think this is better suited for a skin. There's some unfinished work ongoing to allow skins to create settings too, so your whole implementation can be made a skin then.
